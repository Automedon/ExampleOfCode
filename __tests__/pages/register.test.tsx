import renderer from "react-test-renderer";
import React from "react";
import Register from "../../pages/register";

describe("Snapshot register page", () => {
  it("register renders correctly", () => {
    const tree = renderer.create(<Register />);
    expect(tree).toMatchSnapshot();
  });
});
