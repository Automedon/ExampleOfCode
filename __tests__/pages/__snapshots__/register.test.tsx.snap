// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot register page register renders correctly 1`] = `
<div>
  <div>
     
    <h1>
      Not a cool form huh? I will change it later... certainly...
    </h1>
    <form
      action="#"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div>
        Email
         
        <input
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          type="email"
          value=""
        />
      </div>
      <div>
        Password
         
        <input
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
      <div>
        <button
          disabled={false}
          type="submit"
        >
          Submit
        </button>
      </div>
    </form>
  </div>
  <div>
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Home Page
    </a>
  </div>
</div>
`;
